annotation-processor/src/main/kotlin/pl/touk/krush/model/ModelValidator.kt

```diff
@@ -69,7 +71,7 @@ class EntityIdTypeValidator : Validator<EntityDefinition> {
 
     override fun validate(el: EntityDefinition): ValidationResult {
         if (el.id!!.converter == null && el.id.type !in supportedIdTypes) {
-            return Error(listOf(ValidationErrorMessage("Entity ${el.qualifiedName} id type ${el.id.type} is unsupported. Use property converter instead.")))
+            return Error(ValidationErrorMessage("Entity ${el.qualifiedName} id type ${el.id.type} is unsupported. Use property converter instead."))
```

Comment:
\[detekt\] INFO: \[style/MaxLineLength\] Line detected that is longer than the defined maximum line length in the code style\.